docs/_data/features.ts

```diff
@@ -55,12 +56,21 @@ export const mail: FeatureListItem[] = [
 	{ text: "Optional Rainloop installation" },
 ];
 
+export const dns: FeatureListItem[] = [
+	{ text: "Create your own nameservers" },
+	{ text: "Easy setup for a DNS cluster" },
+	{ text: "Support for DNSSEC on domains" },
+];
+
 export const databases: FeatureListItem[] = [
-	{ text: "Support for MariaDB 10.2 -> 10.8 with 10.6 as default" },
+	{ text: "Support for MariaDB 10.2 -> 11.0 with 10.11 as default" },
+	{ text: "Support for Mysql 8" },
 	{ text: "Support for PostgreSQL" },
-	{ text: "Latest version of phpMyAdmin" },
+	{ text: "Latest version of phpMyAdmin and phpPgAdmin" },
 ];
 
 export const serverAdmin: FeatureListItem[] = [
-	{ text: "Automated backups to SFTP, FTP or Backblaze B2" },
+	{
+		text: "Automated backups to SFTP, FTP and via Rclone with 50+ (<a href='https://rclone.org/overview/'>Cloud storage providers</a>",
```

Comment:
It looks better without the parentheses

```suggestion
		text: "Automated backups to SFTP, FTP and via Rclone with 50+ <a href='https://rclone.org/overview/'>Cloud storage providers</a>",
```